src/components/filterSection/constants/inputData.js

```diff
@@ -0,0 +1,25 @@
+import searchIcon from '../../../assests/images/search.svg';
+import favouriteIcon from '../../../assests/images/favourite.svg';
+
+const inputData = [
+    {
+        className : "dish-search", 
+        img : searchIcon, 
+        inputClass : "dish-search", 
+        inputType : "text", 
```

Comment:
By default inputTypes are of text so even it's not present it doesn't matter. But you can keep it. I just told you for KT.